docs/content/release_notes/csi_rn_changelog_1.6.0.md

```diff
@@ -1,3 +1,3 @@
 # 1.6.0 (June 2021)
 
-IBM® block storage CSI driver 1.6.0 adds additional support for Kubernetes 1.21 and Red Hat® OpenShift® 4.8.
\ No newline at end of file
+IBM® block storage CSI driver 1.6.0 added additional support for Kubernetes 1.21 and Red Hat® OpenShift® 4.8.
```

Comment:
consider using something like `includes` to avoid the need of updating this verb in each release

Reply:
includes would also need to be updated. this is how we do it for all of our release notes. leaving as-is. please resolve this comment.

Reply:
IMO "includes" wouldn't need to be updated, because the version will never stop containing its contents, but ok